### PR TITLE
PAdES B-LT example

### PR DESCRIPTION
**Pull request by anovik**
Updated unipdf dependency to 3.50.
Added example for PAdES B-LT.

### CODE REVIEW

**Review by sampila — COMMENTED** [review]
Overall looks good to me, added a comment

File: signatures/pdf_sign_pades_b_lt.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	if err != nil {
+		log.Fatal("Fail: %v\n", err)
+	}
+
+	priv, cert, err := pkcs12.Decode(pfxData, password)
+	if err != nil {
+		log.Fatal("Fail: %v\n", err)
+	}
+
+	// Get cacert certificate from the PEM file.
```
[the remark this answers is not in the document]
Comment (anovik) [reply signatures/pdf_sign_pades_b_lt.go]: Updated the comment

**Review by sampila — APPROVED** [review]
Looks good to me

**Review by 3ace — APPROVED** [review]
lgtm

**Review by gunnsth — APPROVED** [review]

**Pull request merged**